Adjust image labels and add text below images to describe region

sean-fitzpatrick opened the pull request:
Addresses the issue noticed by @grady in #161.

I've added text below each image to describe the region, and I've also adjusted the node shifts for the labels so that they fit in the image, and bit better with the curves than before.

APEXCalculus commented:
@sean-fitzpatrick @Alex-Jordan This looks good to me, but it delves into things I'm not so good at. So I'll let someone else merge this.

Alex-Jordan commented:
It strikes me as not right to have these image descriptions be a `p` that follows the `image`. It's not "right" and also it could (will?) lead to ugly indentation. These are short enough, can they do into a `description` element? Presently that becomes alt text in HTML. There's been some talk to change `description` to some other mechanism (where it can keep markup like math) and have a new `short-description` that could become alt text.

I don't want to just pop in to be a naysayer. I'm on an extended road trip and I can't help with anything substantial. If you need to do something now, I am not sure the current available alternatives would be better.

sean-fitzpatrick commented:
I am fine with putting things in a description, although as you say, currently that won't allow us to use math markup.

The suggestion to put the description in a paragraph might have come from @grady ?

In any case, I've made sure that none of the labels are cut off now, but there needs to be something there to ensure these exercises can be done by someone with visual impairment.

The advantage of a paragraph (at present) is that we can use math, and let MathJax handle accessibility.

sean-fitzpatrick commented:
I've made a quick adjustment to modify the paragraph slightly, and put it above the image. This seems a little better to me?

I forgot to remove a few instances of "In the following exercises". Let me get to that

sean-fitzpatrick commented:
Does anyone have an opinion on whether this is acceptable, at least as a temporary fix?

Throughout Chapter 7, there were exercises where you could only determine the function by looking at the image.
I've fixed the images so that labels are not cut off, so this, at least, is now possible.
But I've also added text preceding the image in each exercise to describe the function/region.

Maybe this should become alt text later, but I think it's OK. I hope to have PDFs up this weekend for inspection.
